index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css" type="text/css" media="all" />
+    <title>Capstone Website</title>
+</head>
+<body class="body">
+  <section class="content">
+    <header id="top-bar">
+      <div class="so-container">
+        <div class="socials">
+            <img id="facebook" src="./images/socials/facebook.svg" alt="Facebook Handle.">
+            <img id="twitter" src="./images/socials/twitter.svg" alt="Twitter Handle.">
+            <ul class="nav-list">
+                <li id="top-1">English</li>
+                <li id="top-2">My Page</li>
+                <li id="top-3">Logout</li>
+            </ul>
+            </div>
+            </div>
+            <nav id="nav-bar">
+              <img id="ham-menu" src="./images/ham.png" alt="Hamburger Menu" />
+               <img id="logo" src="./images/nba-logos/nba-logo.svg" alt="Logo">
+               <img id="exit" src="./images/close-button.png" alt="Exit Menu" />
+               <ul id="my-menu">
+                <li id="menu-1"><a id="about" href="about.html">About</a></li>
+               	<li id="menu-2">Program</li>
+                <li id="menu-3">Join</li>
+                <li id="menu-4">Sponsor</li>
+                <li id="menu-5">News</li>
+                <li id="menu-6">Others</li>
+               </ul>
+            </nav>
+    </header>
+	<section id="welcome">
+        <p id="sub-text">Ladies and Gentlemen</p>
+        <h1 class="wel-title">WELCOME TO THE NBA FINALS<br>2023</h1>
+        <p id="paragraph">After a long season, it's down to the last two teams.Glory awaits those who gets crowned the NBA Champion.</p>
+        <p id="date">2023.06.12 ~ 27</p>
+        <p id="venue">&copy; Chase Center<br><br>&copy; TD Garden.</p>
```

Comment:
- [ ] Your headline section is styled in a bit different way as we have on the design to follow. Try to look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/183117787-61fda828-bb4b-4991-b429-04a9bf4c9269.png)

**Design to follow**
![image](https://user-images.githubusercontent.com/90222110/183117934-7a450bc1-cc9b-41ba-afb9-9984b61705bb.png)

- Starting from the background color, please try to make it whitish so that it can look like the design
- The h1 is very big compared to other text. please correct on that too
- the paragraph text with `id=paragraph` is supposed to have a color somehow dark as it is on the design and somehow small
- PLease try to add some white spaces vertically between your elements in this section

Reply:
- [ ] Please, make sure to implement these changes to your headline, just as the previous reviewer mentioned.